src/styles/blocks/card.scss

```diff
@@ -0,0 +1,113 @@
+.card {
+  width: $card-width;
+  box-sizing: border-box;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+  padding: 32px 16px 16px;
+
+  font-family: Roboto, serif;
+  font-weight: 400;
+
+  transition: all linear $main-during;
+
+  &:hover {
+    transform: scale(1.2);
```

Comment:
The hover effect should smoothly increase the card size by 20% using the correct scale transformation value of `1.2`.